python/rikai/contrib/torch/transforms/ssd.py

```diff
@@ -24,12 +24,7 @@
 
 
 def pre_processing(options: Dict[str, Any]) -> Callable:
-    return T.Compose(
-        [
-            T.Resize([300, 300]),
-            T.ToTensor(),
-        ]
-    )
+    return T.ToTensor()
```

Comment:
SSD model will do resize in the model

Reply:
if we do resizing in pre_processing, the DataLoader will load it with resizing in parallel.

Is the target width or height fixed?

Reply:
If it is fixed for SSD, and it is (300, 300)，why not do benchmark and find the best performant approach?

Reply:
In ssd model, the resize can be done in GPU. 

But the purpose of removing it was not due to a concern of performance.  Letting SSD to handle it (better) can allow us to go through Coco dataset with different size and different number of channels of images.  The current `main` branch (before #529) can not do batch_size > 1 on a full coco dataset scan.